extensions/binding/pom.xml

```diff
@@ -34,6 +34,8 @@
     <module>org.eclipse.smarthome.binding.wemo</module>
     <module>org.eclipse.smarthome.binding.wemo.test</module>
     <module>org.eclipse.smarthome.binding.yahooweather</module>
+    <module>org.eclipse.smarthome.binding.dmx</module>
+    <module>org.eclipse.smarthome.binding.dmx.test</module>
   </modules>
 
-</project>
+</project>
```

Comment:
Could you also add an entry to https://github.com/eclipse/smarthome/blob/master/features/karaf/esh-ext/src/main/feature/feature.xml?